dbt/include/spark/macros/adapters.sql

```diff
@@ -1,19 +1,33 @@
 {% macro file_format_clause() %}
+  {{ return(adapter.dispatch('file_format_clause')()) }}
```

Comment:
For this and the other macros below, could you provide the second argument to `dispatch` (`macro_namespace`) as `'dbt'`?
```suggestion
  {{ return(adapter.dispatch('file_format_clause', 'dbt')()) }}
```

This enables users to install packages with overrides for "global" (core/plugin) macros, and choose to prefer the package's version over the global version by specifying a project config like:
```yml
# dbt_project.yml
dispatch:
  - macro_namespace: dbt
    search_order: ['my_project', 'package_with_custom_overrides', 'dbt']
```

Docs: https://docs.getdbt.com/reference/dbt-jinja-functions/dispatch#overriding-global-macros

(Meta comment: The syntax here has developed organically over time, so it can be a bit clunky. Some thoughts about how we could make it more ergonomic in the future: https://github.com/dbt-labs/dbt-core/issues/4646)

Reply:
Sure, thanks for letting me know! Updated.